extensions/sql-database-projects/src/models/project.ts

```diff
@@ -250,7 +250,8 @@ export class Project {
 	}
 
 	public createProjectEntry(relativePath: string, entryType: EntryType): ProjectEntry {
-		return new ProjectEntry(Uri.file(path.join(this.projectFolderPath, relativePath)), relativePath, entryType);
+		let platformSafeRelativePath = utils.getPlatformSafeFileEntryPath(relativePath);
+		return new ProjectEntry(Uri.file(path.join(this.projectFolderPath, platformSafeRelativePath)), relativePath, entryType);
```

Comment:
Can you also update DatabaseReferenceProjectEntries to have platform safe path? Or I can do it in another PR after this is checked in :) It also has the same problem that it doesn't parse the path correctly on Mac for Windows paths.

Reply:
will do it and send another itr.